src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -160,6 +160,8 @@ public void deleteAppointment(Appointment a) {
         }
 
         @Override
+        public Appointment getAppointment(int index) { throw new AssertionError("This method should not be called."); }
+
```

Comment:
May I know the rational for adding this `getAppointment `method? This seems not used currently, is it from the last implementation?

Reply:
Yes, I will remove this one